lib/matplotlib/axes/_base.py

```diff
@@ -2369,6 +2369,8 @@ def _update_patch_limits(self, patch):
             return
         patch_trf = patch.get_transform()
         updatex, updatey = patch_trf.contains_branch_seperately(self.transData)
+        if not (updatex or updatey):
+            return
```

Comment:
Calling the transform below blows up needlessly because it is called, and then does nothing....